api/controllers/PokemonController.js

```diff
@@ -73,13 +71,7 @@ module.exports = _.mapValues({
 
   async delete (req, res) {
     const id = req.param('id');
-    let pokemon = await Pokemon.findOne({id});
-    if (!pokemon) {
-      return res.notFound();
-    }
-    if (pokemon.owner !== req.user.name && !req.user.isAdmin) {
-      return res.forbidden();
-    }
+    let pokemon = await Validation.verifyUserIsPokemonOwner({user: req.user, id});
```

Comment:
What you are doing is 2 things:

1. Getting the pokemon
2. Checking data against the pokemon

Nothing wrong with having the 2 lines. It is pretty clear code with that, and it's not loads of lines. We should not get a pokemon object from a validator. A validator should just validate something that already exists. Validators should be pure functions.